openvpn/config.sls

```diff
@@ -89,11 +89,17 @@ openvpn_config_{{ type }}_{{ name }}_passwd_file:
       - service: {{ service_id }}
 {% endif %}
 
-{% if config.tls_auth is defined and config.ta_content is defined %}
+{% if config.ta_content is defined %}
 # Deploy {{ type }} {{ name }} TLS key file
+  {% if config.tls_auth is defined and config.tls_crypt is not defined %}
```

Comment:
@alxwr Yes, that would be the better option assuming that the user is using OpenVPN 2.4 and above. Not sure of the best way to incorporate a check if _both_ are defined though. The logic first checks if `ta_content` is defined because it means the user wants to use that static key for either `tls-crypt` _or_ `tls-auth`. Should I change Line 92 to be something like:

Option 1:
`{% if config.ta_content is defined and not (config.tls_crypt is defined and config.tls_auth is defined) %}`

Option 2:
Seperate states for both options
```
{% if config.ta_content is defined and config.tls_crypt is defined %}
openvpn_config_{{ type }}_{{ name }}_tls_crypt_file:
...
{% elif config.ta_content is defined and config.tls_auth is defined %}
openvpn_config_{{ type }}_{{ name }}_tls_auth_file:
...
{% endif %}
```